README.md

```diff
@@ -49,6 +49,37 @@ Each benchmark will run until the target quality is reached and then stop, print
 
 Some these benchmarks are rather slow or take a long time to run on the reference hardware (i.e. 16 CPUs and one P100). We expect to see significant performance improvements with more hardware and optimized implementations. 
 
+
+## Running with Popper
+
+[Popper](https://github.com/systemslab/popper) is a tool for defining and executing container-native workflows either locally or on CI services. We use Popper to convert the benchmarks into reproducible workflows. More details about Popper can be found [here](https://popper.readthedocs.io/).
```

Comment:
"Some workflows in this repository contain a `wf.yml` file that defines a Popper workflow for automatically downloading and verifying data, running the benchmark and generating a report. The execution and report generation both comply with the [MLPerf training rules](https://github.com/mlperf/training_policies/blob/master/training_rules.adoc).